### DISCUSSION TITLE
Remove the limit of number of studies for the study view page

### DISCUSSION THREAD

**Discussion by jjgao** [cBioPortal/cbioportal#9162] [General]
Currently the "Explore Selected Studies" button is disabled when >50 studies are selected. It seems the portal is performant enough to allow view [all curated NR studies (current 188)](https://www.cbioportal.org/study/summary?id=paac_jhu_2014%2Cmel_tsam_liang_2017%2Call_stjude_2016%2Caml_ohsu_2018%2Claml_tcga_pan_can_atlas_2018%2Cmnm_washu_2016%2Cacyc_fmi_2014%2Cacyc_jhu_2016%2Cacyc_mda_2015%2Cacyc_mskcc_2013%2Cacyc_sanger_2013%2Cacc_2019%2Cacbc_mskcc_2015%2Cacc_tcga_pan_can_atlas_2018%2Campca_bcm_2016%2Cbcc_unige_2016%2Cblca_mskcc_solit_2014%2Cblca_mskcc_solit_2012%2Cblca_bgi%2Cblca_dfarber_mskcc_2014%2Cblca_tcga_pan_can_atlas_2018%2Clgg_tcga_pan_can_atlas_2018%2Cbrca_metabric%2Cbrca_mskcc_2019%2Cbrca_smc_2018%2Cbfn_duke_nus_2015%2Cbrca_bccrc%2Cbrca_broad%2Cbrca_sanger%2Cbrca_tcga_pan_can_atlas_2018%2Ccesc_tcga_pan_can_atlas_2018%2Cchol_icgc_2017%2Cchol_nccs_2013%2Cchol_nus_2012%2Cchol_tcga_pan_can_atlas_2018%2Clcll_broad_2013%2Ccll_broad_2015%2Ccll_iuopa_2015%2Ccllsll_icgc_2011%2Cccrcc_dfci_2019%2Ccoad_caseccc_2015%2Ccoad_cptac_2019%2Ccoadread_dfci_2016%2Ccoadread_genentech%2Ccoadread_tcga_pan_can_atlas_2018%2Ccoadread_mskcc%2Chccihch_pku_2019%2Ccscc_dfarber_2015%2Ccscc_hgsc_bcm_2014%2Ccscc_ucsf_2021%2Cctcl_columbia_2015%2Cpact_jhu_2011%2Cdesm_broad_2015%2Cdlbcl_dfci_2018%2Cdlbcl_duke_2017%2Cdlbc_tcga_pan_can_atlas_2018%2Cnhl_bcgsc_2013%2Cesca_broad%2Cesca_tcga_pan_can_atlas_2018%2Cescc_icgc%2Cescc_ucla_2014%2Ces_iocurie_2014%2Cgbc_shanghai_2014%2Cegc_tmucih_2015%2Cstad_oncosg_2018%2Cgbm_columbia_2019%2Cgbm_tcga_pan_can_atlas_2018%2Cglioma_msk_2018%2Chnsc_broad%2Chnsc_jhu%2Chnsc_tcga_pan_can_atlas_2018%2Cliad_inserm_fr_2014%2Chcc_inserm_fr_2015%2Chistiocytosis_cobi_msk_2019%2Cpanet_shanghai_2013%2Cchol_jhu_2013%2Cihch_ismms_2015%2Cihch_smmu_2014%2Ckich_tcga_pan_can_atlas_2018%2Ckirc_bgi%2Cccrcc_irc_2014%2Ckirc_tcga_pan_can_atlas_2018%2Ckirp_tcga_pan_can_atlas_2018%2Chcc_msk_venturaa_2018%2Clihc_amc_prv%2Clihc_riken%2Clihc_tcga_pan_can_atlas_2018%2Clgg_ucsf_2014%2Cluad_broad%2Cluad_oncosg_2020%2Cluad_tcga_pan_can_atlas_2018%2Cluad_tsp%2Clung_smc_2016%2Clusc_tcga_pan_can_atlas_2018%2Cmsk_impact_2017%2Cmixed_allen_2018%2Cmpnst_mskcc%2Cmcl_idibips_2013%2Cmbl_broad_2012%2Cmbl_pcgp%2Cmbl_sickkids_2016%2Cskcm_mskcc_2014%2Cmeso_tcga_pan_can_atlas_2018%2Cbrca_igr_2015%2Cmel_dfci_2019%2Cskcm_dfci_2015%2Cskcm_vanderbilt_mskcc_2015%2Cmel_ucla_2016%2Cprad_mich%2Cprad_su2c_2019%2Cmetastatic_solid_tumors_mich_2017%2Cmm_broad%2Cmds_tokyo_2011%2Cmpn_cimr_2013%2Cnpc_nusingapore%2Cnbl_amc_2012%2Cnbl_ucologne_2015%2Cnepc_wcm_2016%2Cnhl_bcgsc_2011%2Cnsclc_mskcc_2018%2Cnsclc_tracerx_2017%2Cnsclc_unito_2016%2Chnsc_mdanderson_2013%2Cov_tcga_pan_can_atlas_2018%2Cpaad_qcmg_uq_2016%2Cpaad_tcga_pan_can_atlas_2018%2Cpaad_utsw_2015%2Cpanet_jhu_2011%2Cpanet_arcnet_2017%2Call_phase2_target_2018_pub%2Caml_target_2018_pub%2Ces_dfarber_broad_2014%2Cnbl_target_2018_pub%2Cpediatric_dkfz_2017%2Cmixed_pipseq_2017%2Cpptc_2019%2Crt_target_2018_pub%2Cwt_target_2018_pub%2Cpcpg_tcga_pan_can_atlas_2018%2Cplmeso_nyu_2015%2Cpcnsl_mayo_2015%2Cprad_broad%2Cprad_fhcrc%2Cprad_mskcc%2Cprad_eururol_2017%2Cprad_tcga_pan_can_atlas_2018%2Cprad_mskcc_cheny1_organoids_2014%2Cprostate_dkfz_2018%2Cprad_msk_2019%2Cccrcc_utokyo_2013%2Cnccrcc_genentech_2014%2Cmrt_bcgsc_2016%2Crms_nih_2014%2Csummit_2018%2Csarc_mskcc%2Csarc_tcga_pan_can_atlas_2018%2Cskcm_broad%2Cskcm_tcga_pan_can_atlas_2018%2Cskcm_yale%2Cskcm_broad_brafresist_2012%2Cscco_mskcc%2Csclc_jhu%2Csclc_ucologne_2015%2Csclc_cancercell_gardner_2017%2Cvsc_cuk_2018%2Cstad_pfizer_uhongkong%2Cstad_tcga_pan_can_atlas_2018%2Cstad_utokyo%2Ctgct_tcga_pan_can_atlas_2018%2Cangs_painter_2020%2Cangs_project_painter_2018%2Cbrca_mbcproject_wagle_2017%2Cprad_mpcproject_2018%2Ctet_nci_2014%2Cthym_tcga_pan_can_atlas_2018%2Cthca_tcga_pan_can_atlas_2018%2Curcc_mskcc_2016%2Cutuc_mskcc_2015%2Cutuc_cornell_baylor_mdacc_2019%2Cutuc_igbmc_2021%2Cutuc_msk_2019%2Cblca_cornell_2016%2Cucs_jhu_2014%2Cucs_tcga_pan_can_atlas_2018%2Cuccc_nih_2017%2Cucec_tcga_pan_can_atlas_2018%2Cum_qimr_2016%2Cuvm_tcga_pan_can_atlas_2018) now. Should we consider lifting the limit?

We will need to improve how we prioritize which chart to display. This is an existing issue but have more studies will make it more obvious. Maybe we should display only fewer charts by default.

(ps. Mabye we are ready to add pan-study charts in the Quick Search tab for the next step!)

@alisman @inodb @n1zea144 @cBioPortal/product

**Comment by Sjoerd-van-Hagen** [comment]
I have no objections to this but perhaps it would be better to first increase the limit? Especially since we are still adding studies and there may be a point where performance could become problematic again?

The nice think about slowly increasing the limit is that we do not have to go back to lower limits, and it allows for nice tweets ;-)

**Comment by jjgao** [comment]
@Sjoerd-van-Hagen good thinking... I don't think anything below the number of current curated NR studies (188) would be so different/useful from 50 for the public portal, so maybe 200?

**Comment by Sjoerd-van-Hagen** [comment]
I do not have an opinion on what the number should be. Since you guys are paying for the servers, I think it is up to you to decide the limit and provide the infrastructure that can deliver it.
